### PR TITLE
First plot

### PR DESCRIPTION
**Pull request by tillschwoerer**
Hi @MaximDrejdink ,
can you please check!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# gittesting
\ No newline at end of file
+# gittesting
```
Comment (MaximDrejdink) [inline README.md:1]: Could you change this? thx

Comment (MaximDrejdink) [reply README.md:1]: fdsaöfmas

File: airquality.Rmd [hunk 2]
```diff
@@ -40,3 +40,21 @@ df %>%
...
+
+## Plot: NO2 Emissions by day
+```{r}
+df %>% 
+  group_by(wday) %>%
+  summarise(no2 = mean(no2, na.rm=TRUE)) %>%
+  ggplot(mapping = aes(x = wday, y = no2)) +
+  geom_col()
+```
+
```
Comment (MaximDrejdink) [inline airquality.Rmd:59]: pls upload exam results ;)

File: README.md (hunk 1 above)
Comment (tillschwoerer) [reply README.md:1]: I have changed it. Should be fine now.

Comment (tillschwoerer) [reply README.md:1]: After the lecture. OK?

**Pull request merged**